app/components/cron-job.js

```diff
@@ -0,0 +1,30 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  classNames: ['settings-cron'],
+  isDeleting: false,
+  actionType: 'Save',
+  store: Ember.inject.service(),
+
+  disableByBuild: function(key) {
+    var value = '';
+    if (this.get('cron.disable_by_build')) {
+      value = 'Only ';
+    } else {
+      value = 'Even ';
+    }
+    return value + 'if no new commit after last cron build';
+  }.property('cron.disable_by_build'),
+
+  actions: {
+    "delete": function() {
+      if (this.get('isDeleting')) {
+        return;
+      }
+      this.set('isDeleting', true);
```

Comment:
If you do more Ember work in the future, I recommend checking out [`ember-concurrency`](http://ember-concurrency.com/) which is a beautiful addon that simplifies a lot of this ceremony around limiting concurrent tasks and much much more.